internal/backends/python/search_pypi.go

```diff
@@ -0,0 +1,97 @@
+package python
+
+import (
+	"fmt"
+	"net/http"
+	"net/url"
+	"strings"
+
+	"github.com/replit/upm/internal/api"
+	"golang.org/x/net/html"
+)
+
+var WhiteSpaceChars = " \n\t"
+
+// Port of https://github.com/asadmoosvi/pypi-search/blob/main/pypi_search/search.py
+func SearchPypi(query string) ([]api.PkgInfo, error) {
+	endpoint := fmt.Sprintf("http://pypi.org/search/?q=%s", url.QueryEscape(query))
```

Comment:
```suggestion
	endpoint := fmt.Sprintf("https://pypi.org/search/?q=%s", url.QueryEscape(query))
```

I already tested this works.